### PR TITLE
CLI for clearing FQDN proxy cache

### PR DESCRIPTION
**Pull request by nebril**
depends on #6692

Please review only last commit, will require rebase after #6692 is merged

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6745)

### CODE REVIEW

**Comment by nebril** [comment]
test-me-please

**Review by vadorovsky — APPROVED** [review]

**Comment by aanm** [comment]
@nebril there's something odd with the commits in this PR, I almost though this was a backport PR

File: cilium/cmd/fqdn-proxy-cleanup.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var fqdnCleanCacheCmd = &cobra.Command{
+	Use:   "fqdn-clean-cache",
```
Comment (tgraf) [inline cilium/cmd/fqdn-proxy-cleanup.go]: Please define this as multiple levels of subcommands: `cilium fqdn cache clean` We may want to list the cache as well

**Review by tgraf — CHANGES_REQUESTED** [review]

**Comment by nebril** [comment]
Rebased and fixed, thanks for reviews.

**Review by eloycoto — APPROVED** [review]

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
Approving based on only the last commit, since the rest is rebased on top of #6692 .

**Review by tgraf — APPROVED** [review]

**Review by aanm — APPROVED** [review]

**Comment by nebril** [comment]
test-me-please

**Pull request merged**